src/Pages/Detail/Detail.js

```diff
@@ -0,0 +1,273 @@
+import React, { Component} from 'react';
+import DetailInfo from './DetailInfo';
+import './Detail.scss'
+
+class Detail extends Component {
+  constructor(props){
+    super(props);
+    this.state = {
+      count : 1,
+      price: 13000
+    };
+  }
+
+
+  countUp = () => {
+    const { count } = this.state;
+    this.setState({
+      count: count + 1,
+    }
+    )
+  }
+
+  countDown = () => {
+    const { count } = this.state;
+    this.setState({
+      count: count === 1 ? count: count-1,
+    })
+  }
+
+
+  render() {
+    const { price, count } = this.state;
+
+    return (
+      <div className="Detail">
+        <section className="content">
+          <div className="item_photo_info_sec">
+            <div className="item_photo_view_box">
+              <a href="/">
+                <img src="https://store.baemin.com/data/goods/20/11/47/327/327_main_077.png" alt="상품 사진"></img>
+              </a>
+            </div>
+          <div className="item_info_box">
+            <div className="item_detail_tit">
+              <h3>2021 다이어리. 오늘에 집중</h3>
+            </div>
+            <div className="item_detail_list">
+              <div className="item_price">
+                  <div className="price_name">판매가격</div>
+                  <div className="price">3,500원</div>
+              </div>
+              <div className="item_delivery">
+                <div className="delivery_info">배송정보</div>
+                <div className="delivery_price">2,500원  (3만원 이상 구매시 무료)
+                <div className="delivery_time">오후 2시 당일배송마감</div>
+                </div>
+              </div>
+            </div>
+            <div className="item_list_choice">
+              <table>
+                <tbody>
+                  <tr>
+                    <td className="cart_prdt_name">
+                      <div className="delivery_name">
+                        <span>2021 일력 오늘에 집중</span>
+                      </div>
+                    </td>
+                    <td>
+                      <span className="count">
+                        <input 
+                        type="text" 
+                        value={count}
+                        />
```

Comment:
```suggestion
                      <span className="count">
                        <input type="text" value={count} />
```
린트 설정이 잘 되어 있는지 확인 부탁드립니다.

Reply:
넹~